crates/ruma/src/lib.rs

```diff
@@ -7,8 +7,8 @@
 //!
 //! Which crates are re-exported can be configured through cargo features.
 //!
-//! > ⚠ Some details might be missing because rustdoc has trouble with re-exports so you may need
-//!   to refer to other crates' documentations.
+//! > ⚠ Some details might be missing because rustdoc has trouble with re-exports so you may need to
+//! > refer to other crates' documentations.
```

Comment:
```suggestion
//! > ⚠ Some details might be missing because rustdoc has trouble with re-exports so you may need
//! > to refer to other crates' documentations.
```